android/sdl_android/src/main/java/com/smartdevicelink/transport/utl/SdlDeviceListener.java

```diff
@@ -255,44 +250,6 @@ public void notifyConnection(@Nullable VehicleType vehicleType) {
         }
     }
 
-    public static void saveVehicleType(Context context, VehicleType vehicleType, String address) {
```

Comment:
@kostyaBoss There are a lot of Codecov warnings.  Do you need to update or add unit tests?